gnat2goto/driver/range_check.adb

```diff
@@ -66,6 +66,12 @@ package body Range_Check is
       pragma Assert (Success_Insert);
       return Length;
    end Store_Symbol_Bound;
+
+   function Get_Bound_Type (Bound_Index : Integer) return Bound_Type is
```

Comment:
This function appears to be used in the previous commit? Or is this just github being screwy again?

Reply:
No, my bad.